.pre-commit-config.yaml

```
@@ -12,11 +12,11 @@ repos:
       - id: end-of-file-fixer
 
   - repo: https://github.com/commitizen-tools/commitizen
-    rev: v4.6.1
+    rev: v4.7.0
     hooks:
       - id: commitizen
 
   - repo: https://github.com/crate-ci/typos
-    rev: v1.32.0
+    rev: v1
```

```suggestion
    rev: v1.32.0
```

Why does that keep doing that....